.gitignore

```diff
@@ -1,2 +1,5 @@
 /target
 /Cargo.lock
+/tools/__pycache__
+*.swo
+*.swp
```

Comment:
what are those? do we expect users to always have them appear in their local repo :detective:?

Reply:
These Vim swap files can be generated when you kill the vim session. I had around five files in the local src repo before pushing this commit.

Reply:
And this __pycache__ was created while I was doing the `Initial state` procedure in the readme.

Reply:
I think user-specific temporary editor files should be out of the `.gitignore`, as we don't want to support all the possible edge cases; what about `pycache`? is it an artifact of running a testnet?

Reply:
Okay, I'll move Vim swap file extensions to ~/.config/git/ignore file.

Reply:
Done